docs/source/llms/chat-model-intro/index.rst

```diff
@@ -0,0 +1,651 @@
+Tutorial: Getting Started with ChatModel
+========================================
+
+MLflow's :py:class:`mlflow.pyfunc.ChatModel` class provides a standardized way to create production-ready conversational AI models. The resulting models are fully integrated with MLflow's tracking, evaluation, and lifecycle management capabilities. They can be shared with others in the MLflow Model Registry, deployed as a REST API, or loaded in a notebook for interactive use. Furthermore, they are compatible with the widely-adopted OpenAI chat API spec, making them easy to integrate with other AI systems and tools.
+
+If you're already familiar with :py:class:`mlflow.pyfunc.PythonModel`, you might be wondering why :py:class:`mlflow.pyfunc.ChatModel` exists given that you can accomplish the same thing with a custom ``PythonModel``. As GenAI applications become more prevalent and more complex, mapping the inputs, outputs, and parameters between your application and the custom ``PythonModel`` can become quite challenging. The ``ChatModel`` class offers a structured way to define custom conversational AI models that simplifies this process, with a focus on standardized, OpenAI-compatible inputs, outputs, and parameters.
```

Comment:
Another pain point of `PythonModel` for GenAI usecases is that you have to define a fairly complex model signature. While we do have the [infer_signature](https://mlflow.org/docs/latest/python_api/mlflow.models.html#mlflow.models.infer_signature) util, the user needs to define a representative input and output example, which can be just as complex given that there's lots of optional fields.

`ChatModel` does not require you to define a signature since the inputs and outputs are standardized